app/src/main/java/co/nimblehq/compose/crypto/ui/screens/information/CoinInfoSellBuy.kt

```diff
@@ -0,0 +1,65 @@
+package co.nimblehq.compose.crypto.ui.screens.information
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.ui.theme.Color.FireOpal
+import co.nimblehq.compose.crypto.ui.theme.Color.TiffanyBlue
+import co.nimblehq.compose.crypto.ui.theme.ComposeTheme
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp12
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp16
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp20
+import co.nimblehq.compose.crypto.ui.theme.Style.coinInfoSellBuyBackground
+
+@Suppress("FunctionNaming", "LongMethod")
+@Composable
+fun CoinInfoSellBuy(
```

Comment:
How about `SellBuyGroup` or `SellBuyContainer`? 😄

Reply:
@Wadeewee Btw, I think the package name `information` is vague a bit. How about `detail`? 😄 cc @luongvo 

It's not a big deal so I am still fine to leave it as it is.

Reply:
Detail is good to me @hoangnguyen92dn 🚀

Reply:
Detail is good 👍